.github/workflows/rust.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        toolchain: [stable, nightly]
+        toolchain: [stable, nightly, 1.56.1]
```

Comment:
Can it only check for one target? It seems a waste to run 30 github actions.

Reply:
Removed half of them.

Now, it won't build darwin binary on linux, and don't build linux binary on darwin. PTAL @BusyJay